web/src/pages/BookWizard/GeneralPage.module.css

```diff
@@ -0,0 +1,59 @@
+.pageContainer {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.dropZone {
+    height: 450px;
+    width: 450px;
+    float: left;
+    background-color: var(--orange);
+    cursor: pointer;
+    border: 2px solid black;
+    margin-bottom: 25px;
+}
+.dropZone {
+    margin: auto;
+}
+
+.dropZoneContent{
+    height: 100%;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.dropZoneText {
+    composes: h3 from '../../App.css'
+}
+
+.dropZoneImage {
+    width: 100%;
+    height: 100%;
```

Comment:
Can you add `object-fit: cover;` here? Right now if you upload a non-square image, it stretches the image to fit the box. In the app, the image is cropped to fit the box, so we should match that here.